### PR TITLE
Restyle Dropdowns #1201

### PR DESCRIPTION
**Pull request by liaojikunwork**

### CODE REVIEW

**Comment by pimato** [comment]
Only the category/market state picker should have a box-shadow, any other dropdown should not have any shadows:
Bildschirmfoto 2020-10-22 um 20 22 00
Bildschirmfoto 2020-10-22 um 20 21 53

**Comment by pimato** [comment]
can we show the Kleros icon here:
Bildschirmfoto 2020-10-22 um 20 23 26

**Comment by pimato** [comment]
on Hover color should be: #C5CAE9 
active color should be: #9FA8DA

**Review by pimato — CHANGES_REQUESTED** [review]
see comments

**Comment by pimato** [comment]
There is also a bug. Steps do reproduce:

1. Click filter button
2. Click on Arbitration dropdown:
Bildschirmfoto 2020-10-22 um 20 43 01
3. Click on Kleros
4. All appears and widget reloads => BUG: Should be just showing markets with Kleros as an arbitrator.

**Comment by pimato** [comment]
Bildschirmfoto 2020-10-22 um 20 44 51

Curation Source => `Verified by`
All Sources => `Any`
No Sources => `None`

**Comment by pimato** [comment]
The main big dropdowns should have:
`padding: 14px 25px;`

**Comment by pimato** [comment]
Also, for default dropdowns (which are not the big once) I recommend to change to padding from `padding: 9px 13px` to `padding: 9px 16px`

**Comment by liaojikunwork** [comment]
> on Hover color should be: #C5CAE9
> active color should be: #9FA8DA

Screen Shot 2020-10-22 at 8 42 20 PM


@pimato , you mean the items on dropdown list?
Or the borderColor of the dropdown button?

**Comment by liaojikunwork** [comment]
> The main big dropdowns should have:
> `padding: 14px 25px;`

Screen Shot 2020-10-22 at 9 08 16 PM

@pimato , do you mean this one?

**Comment by liaojikunwork** [comment]
> Also, for default dropdowns (which are not the big once) I recommend to change to padding from `padding: 9px 13px` to `padding: 9px 16px`

Screen Shot 2020-10-22 at 9 09 13 PM

@pimato , this one?

**Comment by pimato** [comment]
Bildschirmfoto 2020-10-23 um 11 45 00

Icons should be everywhere 22px in height and width. Also why can´t the item start its padding from the icon height?
Seen this here:
Bildschirmfoto 2020-10-23 um 11 46 05

**Comment by pimato** [comment]
Bildschirmfoto 2020-10-23 um 11 56 38

Can the collapsed widget have the same width as the dropdown button?

**Comment by pimato** [comment]
Bildschirmfoto 2020-10-23 um 12 00 12

- Disconnect should not be active here as it would mean that we are already disconnect.
- missing space between Dropdown button and collapsed widget

**Comment by liaojikunwork** [comment]
> Bildschirmfoto 2020-10-23 um 12 00 12
> 
> * Disconnect should not be active here as it would mean that we are already disconnect.
> * missing space between Dropdown button and collapsed widget

Width of the dropdown should be the same as button?

**Comment by pimato** [comment]
This button should keep its original width. You adopted the width from the dropdown to the button, but should do the opposite instead.
Bildschirmfoto 2020-10-23 um 12 45 02

**Comment by pimato** [comment]
Currency dropdown should have the same height as the verified by dropdown:
Bildschirmfoto 2020-10-23 um 12 46 43

Bildschirmfoto 2020-10-23 um 12 46 46

**Comment by pimato** [comment]
The Dropdown-Button of Currency should display the svg's in 20px exactly like the Verified by Dropdown-Button:

Bildschirmfoto 2020-10-23 um 12 52 15

Bildschirmfoto 2020-10-23 um 12 52 10

Please make sure that the svgs of the items inside the Dropdown do keep the 22px size.

**Comment by pimato** [comment]
A list item of a dropdown should:

- have a reduced padding of `padding: 8px 12px;` to keep the 40px height.
- Icon SVG should be displayed in 24px height.

Bildschirmfoto 2020-10-23 um 13 00 27

**Comment by pimato** [comment]
can we get the All item listed here:
Bildschirmfoto 2020-10-23 um 14 38 18

exactly like we have here:
Bildschirmfoto 2020-10-23 um 14 38 24

**Comment by pimato** [comment]
Inactive items should have a text color of #757575 (textColor in index.js)
Active items should have a text color of37474F (textColorDark in index.js)

**Comment by pimato** [comment]
I am trying to route to `/#/liquidity/curation-source/Any` but it seems it doesn`t find it.

**Comment by pimato** [comment]
I found a bug which disabled both navigation buttons. Steps to reproduce:

1. Click on Filter button
2. Click on Arbitrator Dropdown and pick Kleros
3. Click on Currency and pick Dai
4. Click on Currency and pick All => BUG: now the navigation is showing all buttons disabled. But there are def. more markets.

**Review by pimato — APPROVED** [review]

File: app/src/components/common/form/dropdown/index.tsx [hunk 1]
```diff
@@ -26,12 +26,8 @@ const DropdownOpenCSS = css`
   &,
   &:hover {
```
Comment (KadenZipfel) [inline app/src/components/common/form/dropdown/index.tsx:27]: Maybe we could add a little transition here? @pimato?

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**